ddtrace/appsec/_iast/_ast/ast_patching.py

```diff
@@ -26,23 +26,550 @@
 IAST_DENYLIST: Tuple[Text, ...] = (
     "flask",
     "werkzeug",
+    "aiohttp._helpers",
+    "aiohttp._http_parser",
+    "aiohttp._http_writer",
+    "aiohttp._websocket",
+    "aiohttp.log",
+    "aiohttp.tcp_helpers",
+    "asgiref",
+    "asgiref.current_thread_executor",
```

Comment:
If we add `"asgiref"` to the denylist, we don't need to add the submodules. This applies to `asyncio`, `backports`, `boto3`, `charset_normalizer`, `concurrent` and `django` as well